course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -988,6 +988,38 @@ def _add_course_instructors(self, course_instructors, course_run):
             if not course_run.staff.filter(uuid=instructor.uuid).exists():
                 course_run.staff.add(instructor)
 
+    def _add_course_categories(self, categories, course_run):
+        """
+        Create and add categories to a course run.
```

Comment:
Create and add course subjects to a course run.

Reply:
Done